YoCode/Program.cs

```diff
@@ -1,15 +1,15 @@
 ﻿using System;
+using System.IO;
 
 namespace YoCode
 {
     class Program
     {
         static void Main(string[] args)
         {
-            Console.WriteLine("Welcome to YoCode!");
```

Comment:
Did you deliberately remove the welcome message? Assuming this new feature is intended to output the list of files it found, I would expect it to add that to the existing output, not replace it.